app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -123,13 +133,16 @@ class BookmarksActivity : DuckDuckGoActivity() {
     private fun confirmDeleteBookmark(bookmark: BookmarkEntity) {
         val message = getString(R.string.bookmarkDeleteConfirmMessage, bookmark.title).html(this)
         val title = getString(R.string.dialogConfirmTitle)
-        deleteDialog = AlertDialog.Builder(this)
+        deleteDialog = MaterialAlertDialogBuilder(this)
```

Comment:
because we want to change the background color of the dialogs, we move to `MaterialAlertDialog` and apply a default theme